### PR TITLE
[TEST] Add better test data for detect_breakends

### PR DESCRIPTION
**Pull request by Irallia**
Resolves https://github.com/seqan/iGenVar/issues/33

Change test sam file such that it detect_breakends creates a fasta output aswell.

### CODE REVIEW

File: test/cli/detect_breakends_cli_test.cpp [hunk 1]
```diff
@@ -35,15 +35,24 @@ TEST_F(cli_test, fail_no_argument)
...
     std::string expected
     {
-        "Reference\tchr9\t70103073\tForward\tReference\tchr9\t70103147\tForward\tm13802/6999/CCS\n"
+        "Reference\tchr21\t41972616\tForward\tRead \t0\t2294\tForward\tm2257/8161/CCS\n"
+        "Reference\tchr21\t41972616\tReverse\tRead \t0\t3975\tReverse\tm2257/8161/CCS\n"
+        "Reference\tchr22\t17458417\tForward\tReference\tchr21\t41972615\tForward\tm41327/11677/CCS\n"
+        "Reference\tchr22\t17458418\tForward\tReference\tchr21\t41972616\tForward\tm21263/13017/CCS\n"
+        "Reference\tchr22\t17458418\tForward\tReference\tchr21\t41972616\tForward\tm38637/7161/CCS\n"
     };
     std::string expected_err
```
Comment (simonsasse) [inline test/cli/detect_breakends_cli_test.cpp:48]: Is there already an issue for why this writes to `err` ?

**Review by simonsasse — APPROVED** [review]
Looks good. No empty outputs anymore :)

File: test/cli/detect_breakends_cli_test.cpp (hunk 1 above)
Comment (Irallia) [reply test/cli/detect_breakends_cli_test.cpp:48]: Jepp: https://github.com/seqan/igenvar/issues/23

**Review by Irallia — COMMENTED** [review]
I added a fasta file instead of having the contents of this file in the test file.

**Pull request merged**